consensus/consensus_v2.go

```diff
@@ -15,10 +15,16 @@ import (
 	msg_pb "github.com/harmony-one/harmony/api/proto/message"
 	"github.com/harmony-one/harmony/core"
 	"github.com/harmony-one/harmony/core/types"
+	vrf_bls "github.com/harmony-one/harmony/crypto/vrf/bls"
 	"github.com/harmony-one/harmony/internal/ctxerror"
 	"github.com/harmony-one/harmony/internal/utils"
 	"github.com/harmony-one/harmony/p2p"
 	"github.com/harmony-one/harmony/p2p/host"
+	"github.com/harmony-one/vdf/src/vdf_go"
+)
+
+const (
+	vdfDifficulty = 5000 // This takes about 20s to finish the vdf
```

Comment:
Can make it more than 2 minutes will be better. The current view change timeout for leader is 1 minutes and the leader stays as leader for about 3 minutes

Reply:
ok. this 20s is only for localnet testing purpose. I will update to real delay based on aws instance calibration.

Reply:
used different vdfDifficulty for testnet/mainnet/localnet